skymap_scanner/recos/splinempe_pointed.py

```diff
@@ -0,0 +1,16 @@
+"""IceTray segment for a pointed splinempe reco."""
+
+# mypy: ignore-errors
+ 
+from typing import Final
+
+from . import splinempe 
+from . import RecoInterface       
+
+class SplineMPE_pointed(splinempe.SplineMPE):
```

Comment:
You can call this `SplineMPEPointed` and existing logic should transform the casing from the CL arg